### PR TITLE
fixed issue #269: changed the colour of header bar for light mode in …

### PR DESCRIPTION
**Pull request by Anuragreat**
## PR Description

fixed issue #269: changed the colour of header bar for light mode in mobile view

### CODE REVIEW

File: src/sunpy_sphinx_theme/theme/sunpy/static/sunpy_style.css [hunk 1]
```diff
@@ -60,6 +60,7 @@ html[data-theme="light"] {
   /* A lot of these colors we have inherited from pst however, I have copied them all */
   /* in here to make it obvious which colors are in use, and to shield us from any */
   /* upstream changes. */
+  /* changed the colour on surface for light mode  */
```
Comment (nabobalis) [inline src/sunpy_sphinx_theme/theme/sunpy/static/sunpy_style.css:63]:
```suggestion
```

File: src/sunpy_sphinx_theme/theme/sunpy/static/sunpy_style.css [hunk 2]
```diff
@@ -75,7 +76,7 @@ html[data-theme="light"] {
   --pst-color-link-hover: var(--sst-accent-color-muted);
   --pst-color-link: var(--sst-accent-color-bright);
   --pst-color-on-background: #fff;
-  --pst-color-on-surface: #222832;
+  --pst-color-on-surface: #a2a2a0;
```
Comment (nabobalis) [inline src/sunpy_sphinx_theme/theme/sunpy/static/sunpy_style.css:79]: This change doesnt seem to have worked for me?

Comment (nabobalis) [reply src/sunpy_sphinx_theme/theme/sunpy/static/sunpy_style.css:79]: The issue is the color of the mobile menu icons are black on the light theme

**Comment by Anuragreat** [comment]
Hi..

I seriously am not able to understand the issue, i can find menu icons in
the screenshot you shared on github
Can you share me circled or pointed in screenshot

**Comment by nabobalis** [comment]
Thanks for the PR @Anuragreat but since it didn't work and there has not been any changes since, I will close this. 

If you want to pick it up again, just reopen this.

**Pull request closed without merging**